traffic_stops/templates/home.html

```diff
@@ -34,29 +34,49 @@ <h1 class='title'>Open Data Policing</h1>
     <div class="row">
       <div class="col-md-10 col-md-offset-1">
         <blockquote>
-          <p>"This platform presents the information in a manner that increases the transparency of the Fayetteville Police Department and improves the community and police relationship."</p>
+          <p>“This platform presents the information in a manner that increases
+              the transparency of the Fayetteville Police Department and
+              improves the community and police relationship.”</p>
           <footer>Harold Medlock, Fayetteville Police Chief</footer>
         </blockquote>
       </div>
     </div>
     <hr class="half-rule">
     <div class="row">
       <div class="col-sm-4">
-        <h3>Feature A</h3>
+        <h3>Locate an Officer's Career Stop and Search History</h3>
         <p>
-          Etiam at mollis nisi, a fringilla lorem. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Donec placerat tempor felis, ac ullamcorper orci tincidunt viverra.
+          Stopped by the police? Use our “Find a Stop” feature to locate your
+            traffic stop. Click on the associated Officer ID number to display
+            the enforcement history of the officer who stopped you. Maryland
+            data includes an officer’s stop, search, and contraband seizure data
+            , broken down by race and ethnicity. North Carolina data includes
```

Comment:
put the comma on the previous line; the extra space between data and "," shows up on the page